src/style.css

```diff
@@ -0,0 +1,104 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.cotainer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  align-content: center;
+}
+
+.block {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  flex-basis: 300px;
+  height: 300px;
+  min-width: 300px;
+  font-size: 100px;
+  color: white;
+}
+
+.colored__block1 {
+  background-color: rgb(255, 0, 0);
+}
+
+.colored__block2 {
+  background-color: rgb(204, 0, 0);
+}
+
+.colored__block3 {
+  background-color: rgb(153, 0, 0);
+}
+
+.colored__block4 {
+  background-color: rgb(102, 0, 0);
+}
+
+.colored__block5 {
+  background-color: rgb(51, 0, 0);
+}
+
+.colored__block6 {
+  background-color: rgb(0, 0, 0);
+}
+
+@media(min-width: 1200px) {
+  .cotainer {
+    padding: 0  150px;
+  }
+}
+
+@media(min-width: 600px) {
+  .cotainer {
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  .colored__block1 {
+    order: -5;
+  }
+
+  .colored__block2 {
+    order: -5;
+  }
```

Comment:
two blocks with the same order
I think you can change the order for one of them